docs/sites.yml

```diff
@@ -10012,3 +10012,29 @@
   built_by: Rašid Redžić
   built_by_url: https://rasidre.com/
   featured: false
+- title: Creative code daily
+  main_url: https://www.bobylito.dev/
+  url: https://www.bobylito.dev/
+  source_url: https://github.com/bobylito/sketches
+  description: >
+    Creative code daily (CCD) is a personal project for which I build a new animation made out of code every day.
+  categories:
+    - Blog
+    - Programming
+    - Gallery
+    - Portfolio
+  built_by: Alexandre Valsamou-Stanislawski
+  built_by_url: https://www.noima.xyz
+- title: Playlist Detective
```

Comment:
It looks like you've already added this site before? Could we remove this entry so it isn't represented twice and instead edit the metadata for the first one? 🙂 

https://github.com/gatsbyjs/gatsby/blob/e1741584f575b5fcd8225ce9eab4602d77e2c3ab/docs/sites.yml#L9916

Reply:
Yes of course. Sorry about that.